bin/dfx-ii-create-dummy-anchor

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+SOURCE_DIR="$(dirname "$(realpath "${BASH_SOURCE[0]}")")"
+
+print_help() {
+  cat <<-EOF
+
+	Creates a dummy internet identity with a registered backup device and seed 
+  phrase. This makes it easy to sign in with the fewer number of clicks.
+	EOF
+}
+
+# Source the clap.bash file ---------------------------------------------------
+source "$SOURCE_DIR/clap.bash"
+# Define options
+clap.define short=n long=network desc="The dfx network to use" variable=DFX_NETWORK default="local"
+clap.define short=i long=identity desc="The dfx identity to use" variable=DFX_IDENTITY default="$(dfx identity whoami)"
+# Source the output file ----------------------------------------------------------
+source "$(clap.build)"
+
+IDENTITY_PUB_KEY_FILE="$(mktemp)"
+
+clean_up() {
+  rm "$IDENTITY_PUB_KEY_FILE"
+}
+
+trap clean_up EXIT
+
+DECLARATIONS_DIR="$(realpath "$SOURCE_DIR/../declarations")"
+II_DID_FILE="$DECLARATIONS_DIR/internet_identity.did"
+
+dfx identity export "$DFX_IDENTITY" | openssl ec -out "$IDENTITY_PUB_KEY_FILE" -pubout -outform der 2>/dev/null
+
+echo "Fetching captcha."
+CHALLENGE_KEY="$(dfx canister call internet_identity create_challenge --candid "$II_DID_FILE" --network "$DFX_NETWORK" | idl2json | jq -r .challenge_key)"
+
+PUB_KEY_BLOB=$(hexdump -ve '1/1 "%.2x"' "$IDENTITY_PUB_KEY_FILE" | sed 's/../\\&/g')
+
+echo "Creating anchor."
+ANCHOR="$(dfx canister call internet_identity register "(record{pubkey=blob \"${PUB_KEY_BLOB}\";alias=\"dfx test key\";purpose=variant{authentication};key_type=variant{unknown};protection=variant{unprotected};},record{key=\"${CHALLENGE_KEY}\";chars=\"a\"})" --candid "$II_DID_FILE" --network "$DFX_NETWORK" | idl2json | jq -r .registered.user_number)"
+
+II_CANISTER_ID="$(dfx canister id internet_identity --network "$DFX_NETWORK")"
+
+# This matches the dummy key pair generated by Internet Identity here:
+# https://github.com/dfinity/internet-identity/blob/1ce860dd0601312f6521d433c4e273215a1abc5d/src/frontend/src/utils/iiConnection.ts#L57
+DUMMY_PUB_KEY_BLOB='0*0\05\06\03+ep\03!\00;j\27\bc\ce\b6\a4-b\a3\a8\d0*o\0dse2\15w\1d\e2C\a6:\c0H\a1\8bY\da)'
+DUMMY_CREDENTIAL_ID_BLOB='\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00'
```

Comment:
The credential ID is `opt` and could be omitted.

Reply:
Removed.